site/_posts/2017-01-23-last-week-in-manageiq-last-will-be-first.html.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Last Week in ManageIQ: The last will be the first!"
+author: hstastna
+date: 2017-01-23
+tags: LWIMIQ
+comments: true
+published: true
+---
+
+Hi all! [Hilda](https://github.com/hstastna) here. This is my first Last Week in ManageIQ post, I hope that not my last! Below is a review of the last week's changes.
+
+## Featured
+
+As we are getting used to splitting repos, deleting so many lines and preparing for Angular 2 in the UI, there were 97 [PRs] merged last week in manageiq core, more than the week before, containing 226 [commits] by 43 [contributors]. You can choose from our list of [open issues], if you would like to contribute. Any help is appreciated.
+
+On Wednesday, we participated at the sprint review meeting as usually, as it was the end of Sprint 52 on Monday, January 16th. You can find the recorded video in the [ManageIQ channel](https://www.youtube.com/watch?v=eVuhBZ53SP0). By the way, 52 is quite nice number. 52 = 2^2 x 13, and about two weeks ago, we had Friday 13th :)
```

Comment:
Instead of "we participated at the sprint review meeting as usually, as it was the end of Sprint 52 on Monday, January 16th.",..perhaps use, "we participated in the sprint review meeting as usual", as this was for the end of Sprint 52 for Monday, January 16th."